packages/components/src/templates/next/components/internal/CollectionCard/CollectionCard.tsx

```diff
@@ -47,6 +49,13 @@ export const CollectionCard = ({
             <span title={itemTitle}>{itemTitle}</span>
           </Link>
         </h3>
+        {tags && tags.length > 0 && (
+          <div className="flex flex-wrap gap-2">
```

Comment:
question - figma shows gap 6px which should be gap-1.5?

<img width="1062" alt="image" src="https://github.com/user-attachments/assets/30fcd3ad-a5f0-48d3-bd99-f484851c1dd5">

Reply:
good catch, updated!